src/lib.rs

```diff
@@ -1985,9 +1998,9 @@ pub unsafe trait FromBytes: FromZeros {
     #[inline]
     fn ref_from_suffix(bytes: &[u8]) -> Option<&Self>
```

Comment:
Need to figure out how to reword this paragraph from the doc comment:

```rust
    /// `ref_from_suffix` returns a reference to the last `size_of::<Self>()`
    /// bytes of `bytes`. If `bytes.len() < size_of::<Self>()` or the suffix of
    /// `bytes` is not aligned to `align_of::<Self>()`, this returns `None`.
```

Reply:
Added a TODO in #29 to block release on updating this doc comment, so we don't have to resolve this now.